### ISSUE TITLE
Feature Request: Add ability to format links in chat text (a la markdown) in chat client.

### ISSUE THREAD

**Issue by brad-harper** [keybase/keybase-issues#3354]
Formatting chat text is very limited. Adding the ability to format links, in the manner of markdown, would be helpful. E.g.

```
   [a link to google](https://www.google.com)
```

**Comment by edmundlaugasson** [comment]
Would be also interested and to be honest - I am not alone.
Actually markdown is cool but too freaky for some people.
Making possible to use shortcuts could facilitate formatting a lot:
- CTRL+I _italic_
- CTRL+B **bold**
- CTRL+U underline
- CTRL+K [hyperlink](https://keybase.io/)
- CTRL+SHIFT+M monospace
- etc.
Possibly also some other formatting, e.g. see [Discord ones](https://support.discordapp.com/hc/en-us/articles/210298617-Markdown-Text-101-Chat-Formatting-Bold-Italic-Underline-), [Slack formatting](https://get.slack.help/hc/en-us/articles/202288908-Format-your-messages), [Telegram formatting](https://www.techmesto.com/telegram-desktop-now-supports-text-formatting-using-keyboard-shortcuts/) etc.

**Comment by entropee** [comment]
MatterMost markdown is the best of class.

**Comment by edmundlaugasson** [comment]
In addition there would be nice to see small formatting toolbar - could be floating one like in many messaging apps.

**Comment by Hexstream** [comment]
> Formatting chat text is very limited. Adding the ability to format links, in the manner of markdown, would be helpful. E.g.
> 
> ```
>    [a link to google](https://www.google.com)
> ```

This would be a really great feature to have for channel descriptions, too.
Full URLs there are super bulky.

**Comment by johnkitchen** [comment]
I'd love to see this functionality as well.

**Comment by HaroldMills** [comment]
I'd like this, too. I just wanted to do it, and a Google search for instructions led me here.

**Comment by roguepullrequest** [comment]
> 
> 
> I'd like this, too. I just wanted to do it, and a Google search for instructions led me here.

Yeah can say the same. Just seems natural option to have on something like keybase chat.

**Comment by TRGoCPftF** [comment]
Has any primary maintainers looked into this?

I can't imagine this would be too time consuming of a PR to take on when I get a second if no one is already actively working on this/already planned for implementation in a future build. 

As I'm also dying for this feature, enough to go through and attempt to implement it XD 

Only concern would be ensuring there's a means for hovering to expose full link, or having a warning popup to confirm redirection that displays the full URL akin to what discord does because _"Links are Spoopy"_ and ensure it's clear and easy for users to be able to review the full link before one takes it.

**Comment by Hexstream** [comment]
Keybase got acqui-hired by Zoom, so I would expect a slowdown in new features, at least.

**Comment by jeury301** [comment]
hey, is this resolved?

**Comment by navjotjsingh** [comment]
Looks like Zoom bought this just because it was under attack for privacy issues and nothing more. Keybase is dead IMO.
